recipes/pocketsphinx/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pocketsphinx-python" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bambocher/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
could we get this from pypi instead?

Reply:
Sure.  When I first looked at [PyPI](https://pypi.org/project/pocketsphinx/#files), I only saw the `wheel` files, but upon closer inspection, I see the source tarball in there.